### PR TITLE
Add support for WeTek Play 2 TV box

### PR DESCRIPTION
**Pull request by codesnake**
Add a project for WeTek Play 2 Amlogic S905 based TV box with DVB-S2/T/T2/C tuners (optional). Both eMMC and SD-card installations are supported.

### CODE REVIEW

**Comment by lrusak** [comment]
Looks pretty good, can we move any of the Linux patches to our linux-amlogic tree?

**Comment by codesnake** [comment]
@lrusak DTS patch will be moved for sure. Possibly DVB patches too.

File: projects/WeTek_Play_2/install/updater-script [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+write_raw_image(package_extract_file("recovery.img"), "recovery");
+
+show_progress(0.018000, 0);
+
+ui_print("Writing bootloader");
+write_raw_image(package_extract_file("bootloader.img"), "bootloader");
+
+set_bootloader_env("upgrade_step", "1");
+show_progress(0.100000, 0);
+ui_print("OpenELEC Installed Successfully");
```
Comment (chewitt) [inline projects/WeTek_Play_2/install/updater-script]: ^^ freudian slip :)

File: projects/WeTek_Play_2/options [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    OPTIMIZATIONS="size"
+
+  # Project CFLAGS
+    PROJECT_CFLAGS=""
+
+  # LTO (Link Time Optimization) support
+    LTO_SUPPORT="yes"
+
+  # GOLD (Google Linker) support
+    GOLD_SUPPORT="yes"
```
Comment (chewitt) [inline projects/WeTek_Play_2/options]: some items here are not required after https://github.com/LibreELEC/LibreELEC.tv/commit/e2dd1e7bd100a0477f0499098c6db98ef5404901

**Comment by lrusak** [comment]
Please rebase and bump amlogic kernel version when you get a chance :smile:

File: projects/WeTek_Play_2/install/updater-script (hunk 1 above)
Comment (chewitt) [inline projects/WeTek_Play_2/install/updater-script]: You can fix that at the same time then!

**Comment by codesnake** [comment]
Rebased the PR: updated Amlogic kernel to 3.14-eb67324, fixed issues pointed by @chewitt, also added serial-console.service, same as recently added by @kszaq to WeTek Hub, WeTe Core and WeTek Play projects.

**Comment by stefansaraev** [comment]
@codesnake fyi. to make refreshrate switch work properly while playing 4k material on 4k display I had to replace your 0001-aml-Fix-video-rect-calculation-when-using-GUI-scalin.patch with https://github.com/stefansaraev/xbmc/commit/77008ead0e4319517adc942b13019d2d74f79646 (part of #772)

**Comment by codesnake** [comment]
Rebased once again: replaced video scaling patch with @wrxtasy's patch (thanks @stefansaraev for pointing out), updated header of platform_init script.

**Comment by lrusak** [comment]
thanks! :+1:

**Pull request merged**